link-grammar/string-set.h

```diff
@@ -27,4 +28,28 @@ const char * string_set_add(const char * source_string, String_set * ss);
 const char * string_set_lookup(const char * source_string, String_set * ss);
 void         string_set_delete(String_set *ss);
 
+/**
+ * Compare 2 strings, assuming they are in the same string-set.
+ * Return true if they are equal, else false.
```

Comment:
Please add a note explaining the "obvious": two string-set strings are equal if and only if their pointers are equal.  Otherwise, one foggy-headed day in the future, the == compare will be very  ... confusing.